File: src/main/java/br/com/southsystem/assembleia/repositorio/PautaRepository.java
package br.com.southsystem.assembleia.repositorio;

import java.util.List;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import br.com.southsystem.assembleia.entidade.Pauta;

@Repository
public interface PautaRepository extends JpaRepository<Pauta, Long> {

	List<Pauta> findByFlFinalizadaFalse();

}

File: src/main/kotlin/br/com/southsystem/desafiobackvotos/gateway/database/PautaRepository.kt
package br.com.southsystem.desafiobackvotos.gateway.database

import br.com.southsystem.desafiobackvotos.domain.Pauta
import org.springframework.data.repository.reactive.ReactiveCrudRepository

interface PautaRepository: ReactiveCrudRepository<Pauta, Long> {
}